assets/views/home/index.ejs

```diff
@@ -104,7 +104,7 @@
         </div>
       </form>
 
-      <%- partial('references.ejs', {theUser: "user", refPage: false}) %>
+      <%- partial('references.ejs', {theUser: "refUser", refPage: false}) %>
```

Comment:
Well, "theUser" is unnecessary if "refUser" is always going to be the user we use now.